<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Equilibreuse</class>
 <widget class="QMainWindow" name="Equilibreuse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1396</width>
    <height>805</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Equilibreuse</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>9</y>
      <width>341</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupeBoxOrigine">
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>120</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>120</height>
        </size>
       </property>
       <property name="title">
        <string>Position origine</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLCDNumber" name="lcdNumberOrigine"/>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="buttonPrendreOrigine">
            <property name="text">
             <string>Prendre
l'origine</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QGroupBox" name="groupeBoxConsigneVitesse">
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>390</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>280</width>
         <height>390</height>
        </size>
       </property>
       <property name="title">
        <string>Consigne de vitesse (en %)</string>
       </property>
       <widget class="QPushButton" name="buttonRazVitesse">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>310</y>
          <width>271</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>RAZ vitesse moyenne</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>271</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutVitesseRotation">
         <item>
          <widget class="QLabel" name="label_VitesseRotation">
           <property name="text">
            <string>Vitesse de rotation :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVitesseMoteur"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Unite1">
           <property name="text">
            <string>trs/min   </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>250</y>
          <width>271</width>
          <height>29</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayoutVitesseMoyenne">
         <item>
          <widget class="QLabel" name="label_VitesseMoyenne">
           <property name="text">
            <string>Vitesse moyenne :   </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditVitesseMoyenne"/>
         </item>
         <item>
          <widget class="QLabel" name="label_Unite2">
           <property name="text">
            <string>trs/min   </string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>271</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttonLancerMoteur">
           <property name="text">
            <string>Lancer Moteur</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonArretMoteur">
           <property name="text">
            <string>Arrêter Moteur</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>271</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QDial" name="buttonFixerConsigneVitesse">
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLCDNumber" name="lcdNumberPourcentageVitesse"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QWidget" name="widgetImage" native="true">
       <property name="minimumSize">
        <size>
         <width>290</width>
         <height>170</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>290</width>
         <height>170</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">Border-image: url(:/images/Maquette.png);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widgetCourbes" native="true">
    <property name="geometry">
     <rect>
      <x>348</x>
      <y>139</y>
      <width>890</width>
      <height>609</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>890</width>
      <height>600</height>
     </size>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupeBoxAcquisition">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>890</width>
      <height>130</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>890</width>
      <height>130</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>890</width>
      <height>130</height>
     </size>
    </property>
    <property name="title">
     <string>Acquisition</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>861</width>
       <height>92</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QTableWidget" name="tableWidgetAcquisition">
        <property name="minimumSize">
         <size>
          <width>455</width>
          <height>85</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>455</width>
          <height>85</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="tabKeyNavigation">
         <bool>false</bool>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="dragDropOverwriteMode">
         <bool>false</bool>
        </property>
        <row>
         <property name="text">
          <string>Palier A</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignLeading|AlignVCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
        </row>
        <row>
         <property name="text">
          <string>Palier O</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="foreground">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Effort Max (N)</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Déphasage (°)</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Effort à 0°   (N)</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Effort à 90° (N)</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
        </column>
        <item row="0" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="0" column="1">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="0" column="2">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="0" column="3">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>255</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="1" column="0">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="1" column="1">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="background">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>0</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="1" column="2">
         <property name="text">
          <string>0</string>
         </property>
         <property name="toolTip">
          <string notr="true"/>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </item>
        <item row="1" column="3">
         <property name="text">
          <string>0</string>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="textAlignment">
          <set>AlignCenter</set>
         </property>
         <property name="foreground">
          <brush brushstyle="NoBrush">
           <color alpha="255">
            <red>255</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLancerAcquisition">
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>90</height>
         </size>
        </property>
        <property name="text">
         <string>Figer
Acquisition</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxVitesse">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>90</height>
         </size>
        </property>
        <property name="title">
         <string>Vitesse de l'expérience </string>
        </property>
        <widget class="QWidget" name="layoutWidget_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>171</width>
           <height>30</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEditVitesseExperience">
            <property name="maximumSize">
             <size>
              <width>160</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_Unite1_2">
            <property name="text">
             <string>trs/min   </string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1396</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionNouvelle_Experience"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="separator"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuExperience">
    <property name="title">
     <string>Expérience</string>
    </property>
    <widget class="QMenu" name="menuExporter">
     <property name="title">
      <string>Exporter</string>
     </property>
     <addaction name="actionMesures_brutes"/>
     <addaction name="actionFormat_CSV"/>
     <addaction name="actionFormat_PDF"/>
    </widget>
    <widget class="QMenu" name="menuOption">
     <property name="title">
      <string>Option</string>
     </property>
     <addaction name="actionAffichage_en_degre"/>
     <addaction name="actionAffichage_en_Newton"/>
     <addaction name="actionDecalage_origine"/>
    </widget>
    <addaction name="actionImporter_mesures_brutes"/>
    <addaction name="menuExporter"/>
    <addaction name="separator"/>
    <addaction name="actionConfigurer"/>
    <addaction name="menuOption"/>
   </widget>
   <widget class="QMenu" name="menuEquilibreuse">
    <property name="title">
     <string>Equilibreuse</string>
    </property>
    <addaction name="actionParametrer"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuExperience"/>
   <addaction name="menuEquilibreuse"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionQuitter"/>
   <addaction name="actionNouvelle_Experience"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="separator"/>
   <addaction name="actionFormat_PDF"/>
   <addaction name="actionFormat_CSV"/>
   <addaction name="separator"/>
   <addaction name="actionParametrer"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNouvelle_Experience">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Nouveau.png</normaloff>:/images/Nouveau.png</iconset>
   </property>
   <property name="text">
    <string>Nouvelle expérience</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Ouvrir.png</normaloff>:/images/Ouvrir.png</iconset>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Enregistrer.png</normaloff>:/images/Enregistrer.png</iconset>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionImporter_mesures_brutes">
   <property name="text">
    <string>Importer mesures brutes</string>
   </property>
  </action>
  <action name="actionMesures_brutes">
   <property name="text">
    <string>Mesures brutes</string>
   </property>
  </action>
  <action name="actionFormat_CSV">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Excel.png</normaloff>:/images/Excel.png</iconset>
   </property>
   <property name="text">
    <string>Format CSV</string>
   </property>
  </action>
  <action name="actionFormat_PDF">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/LogoPdf.png</normaloff>:/images/LogoPdf.png</iconset>
   </property>
   <property name="text">
    <string>Format PDF</string>
   </property>
  </action>
  <action name="actionConfigurer">
   <property name="text">
    <string>Configurer</string>
   </property>
  </action>
  <action name="actionAffichage_en_degre">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Affichage en degré</string>
   </property>
  </action>
  <action name="actionAffichage_en_Newton">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Affichage en Newton</string>
   </property>
  </action>
  <action name="actionDecalage_origine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Décalage origine</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Croix.png</normaloff>:/images/Croix.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionParametrer">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/Parmetres.png</normaloff>:/images/Parmetres.png</iconset>
   </property>
   <property name="text">
    <string>Paramétrer</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>Equilibreuse</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
